website/docusaurus.config.js

```diff
@@ -74,8 +74,8 @@ var siteSettings = {
     },
     announcementBar: {
       id: "monthly-demos",
-      content: "Join our monthly demos and see dbt Cloud in action!",
-      backgroundColor: "#047377",
+      content: "Join us for dbt Developer day on March 19th and March 20th for exciting new and coming-soon features to supercharge data developer workflows!",
+      backgroundColor: "#DE5D43",
```

Comment:
do we like the orange or should we stick w the traditional dbt green?

Reply:
I was informed that the orange is not good for accessibility. In fact, I need to change the table headers from orange to an approved color, this was a good reminder. Suggest we stick with the green